### ISSUE TITLE
[Suggestion]

### ISSUE THREAD

**Issue by kvllx** [Maxx53/SCMBot#292]
This is my suggestion, some people maybe don't want to buy immediately an item, but would like to get some sort of alert when some item reaches X price. Right now we get red allert but no sound.
For example i want to buy awp under 80 bucks, i would like to set the alarm to announce me via  sound alarm when that awp is 80bucks or under 80bucks or any other price selected by me. The alarm sound should be a calm sound and with the option to disable/post pone it for a X amount of time or other price drop.

Another usefull suggestion would have to be a server side ( your side ) , a computer that scans each single item and alerts us(if we want )  ( our computers ) if ITEM X it's price dropped let's say 30% or any percent you like. That would save us a lot of ram/network usage, although this would be expensive for you.

Make an option to sell the items, only if the prices hit a certain level . Let's say the bot buys crates for 0.03 but it doesn't sell them  until de market price reaches 0.10  ( market price set by me ) .

Make the bot to make a sound when he is buying/selling an item, make it register it in the log file.

Make the to log how many items he bought/sold, money spent/money profit, that day ( session )  . This would be extremely useful.
Good luck!!!

If you ever consider making a V.I.P bot , i will buy it .

**Comment by deltaformation** [comment]
I personally think you have no clue wtf you're talking about.

1st,  your alert system already exists in other web api's that you can use.
2nd, you want a feature with Maxx53 hosting a server etc etc so you get the prices...sigh, retarded idea.
3rd you can already choose the price to sell item at..
4th there is already a sound when you buy items, and not required when you send sell orders..
5th the log feature you stated can be made, but why don't you make it yourself and commit to this git ?
6th you have no clue what you're talking about, vip bot... this is github, want vip bot go buy a rip off of this free ***github***project for 20 euros.


TLDR: want a feature, commit and put it yourself.
